File: Programming Basics/Conditional Statements Advanced - Exercises/06.OperationsBetweenNumbers.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        double firstNumber = double.Parse(Console.ReadLine());
        double secondNumber = double.Parse(Console.ReadLine());
        char operation = char.Parse(Console.ReadLine());

        double result;

        switch (operation)
        {
            case '+':
                result = firstNumber + secondNumber;
                if (result % 2 == 0)
                    Console.WriteLine($"{firstNumber} + {secondNumber} = {result} - even");
                else
                    Console.WriteLine($"{firstNumber} + {secondNumber} = {result} - odd");
                break;
            case '-':
                result = firstNumber - secondNumber;
                if (result % 2 == 0)
                    Console.WriteLine($"{firstNumber} - {secondNumber} = {result} - even");
                else
                    Console.WriteLine($"{firstNumber} - {secondNumber} = {result} - odd");
                break;
            case '*':
                result = firstNumber * secondNumber;
                if (result % 2 == 0)
                    Console.WriteLine($"{firstNumber} * {secondNumber} = {result} - even");
                else
                    Console.WriteLine($"{firstNumber} * {secondNumber} = {result} - odd");
                break;
            case '/':
                result = firstNumber / secondNumber;
                if (secondNumber == 0)
                    Console.WriteLine($"Cannot divide {firstNumber} by zero");
                else
                    Console.WriteLine($"{firstNumber} / {secondNumber} = {result:f2}");               
                break;
            case '%':
                result = firstNumber % secondNumber;
                if (secondNumber == 0)
                    Console.WriteLine($"Cannot divide {firstNumber} by zero");
                else
                    Console.WriteLine($"{firstNumber} % {secondNumber} = {result}");
                break;
        }
    }
}

File: Programming Basics - Python/Conditional Statements Advanced - Exercises/06.OperationsBetweenNumbers.py
first_number = int(input())
second_number = int(input())
operation = input()

if second_number == 0 and operation in ['/', '%']:
    print(f'Cannot divide {first_number} by zero')
    exit()

result = eval(f'{first_number} {operation} {second_number}')

print(f'{first_number} {operation} {second_number} = ', end='')

if operation in ['+', '-', '*']:
    print(f"{result} - {'even' if result % 2 == 0 else 'odd'}")

elif operation == '/':
    print(f'{result:.2f}')

elif operation == '%':
    print(f'{result}')